wrappers/src/shared_realm_cs.cpp

```diff
@@ -68,7 +68,7 @@ REALM_EXPORT SharedRealm* shared_realm_open(Schema* schema, uint16_t* path, size
 
         Realm::Config config;
         config.path = pathStr.to_string();
-        config.read_only = read_only;
+        //config.read_only = read_only;
```

Comment:
I see replacement code a few lines below so this looks like cruft that should have just been deleted.